src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/mappers/UuidMapper.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.controller.mappers
+
+import org.springframework.web.server.ServerWebInputException
+import java.util.UUID
+
+fun parseID(id: String): UUID {
```

Comment:
this isn't necessary - see the changes in https://github.com/ministryofjustice/hmpps-interventions-service/pull/113/commits/ab44532b6635cdfdbae18b989a6bfbddeeb07bb5 which remove this and just allow spring/jackson to parse these UUIDs. no need to pull this out though, i'll rebase my changes onto this once it's merged.